core/src/main/java/kin/ecosystem/core/util/Converter.java

```diff
@@ -0,0 +1,46 @@
+package kin.ecosystem.core.util;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import kin.ecosystem.common.model.NativeOffer;
+import kin.ecosystem.common.model.NativeSpendOffer;
+import kin.ecosystem.core.network.model.Offer;
+import kin.ecosystem.core.network.model.Offer.ContentTypeEnum;
+import kin.ecosystem.core.network.model.Offer.OfferType;
+
+public class Converter {
```

Comment:
why it's needed in this PR?

Reply:
because NativeOffer does not extends Offer anymore.